frontstage/views/secure_messaging/create_message.py

```diff
@@ -77,3 +77,26 @@ def send_message(party_id, is_draft, case_id, survey, ru_ref):
     logger.info('Secure message sent successfully',
                 message_id=sent_message['msg_id'], party_id=party_id)
     return sent_message
+
+
+def create_headers():
+    encoded_jwt = SessionHandler().get_encoded_jwt(request.cookies['authorization'])
+    headers = {"jwt": encoded_jwt}
+    return headers
+
+
+def get_message(message_id, label, party_id):
+    logger.debug('Attempting to retrieve message', message_id=message_id, party_id=party_id)
```

Comment:
I think this is a good info level, give me all teh information about the message id and party id.

Reply:
What do you mean give you all the information?  That's literally all the information this function gets, other than label, but it doesn't feel worthwhile logging that out.